src/libraries/Microsoft.Extensions.Options/tests/SourceGenerationTests/Baselines/NetCoreApp/Validators.g.cs

```diff
@@ -73,11 +89,19 @@ partial class SecondValidatorNoNamespace
         var baseName = (string.IsNullOrEmpty(name) ? "SecondModelNoNamespace" : name) + ".";
         var builder = new global::Microsoft.Extensions.Options.ValidateOptionsResultBuilder();
         var context = new global::System.ComponentModel.DataAnnotations.ValidationContext(options);
+        var validationResults = new global::System.Collections.Generic.List<global::System.ComponentModel.DataAnnotations.ValidationResult>();
```

Comment:
Just like the attributes are allocated as statics, could we just create these lists statically as well and reuse them?

Or at the very least, the lists could be allocated with an explicit capacity since this is known by the generator.